stac_fastapi/pgstac/setup.py

```diff
@@ -16,6 +16,7 @@
     "asyncpg",
     "buildpg",
     "brotli_asgi",
+    "psycopg[binary]",
```

Comment:
I think this needs to be added to pypgstac dependencies.

Reply:
https://github.com/stac-utils/pgstac/issues/105

Reply:
I don't think we need psycopg in stac-fastapi.pgstac because we are using asyncpg

Reply:
It's now been moved to pypgstac and only included as a dev dependency here to allow pypgstac to manage the unit test database in conftest.py.